src/integrations/feed/clients.ts

```diff
@@ -37,10 +43,19 @@ export class FeedClient implements IFeedClient {
       return { data: [] };
     }
     return {
-      data: res.data.map(({ post_id, metadata }) => [
-        post_id,
-        (metadata && JSON.stringify(metadata)) || null,
-      ]),
+      data: res.data.map(({ post_id, metadata }) => {
+        const hasMetadata = !!(metadata || extraMetadata);
+
+        return [
+          post_id,
+          (hasMetadata &&
+            JSON.stringify({
+              ...metadata,
+              mab: extraMetadata,
```

Comment:
Originally this is what I meant
```suggestion
              ...extraMetadata,
```

Reply:
Ahh yes, will update.